flycheck.el

```diff
@@ -8324,6 +8325,33 @@ See URL `http://www.scalastyle.org'."
                        ((not config-file) '(bold error))
                        (t 'success)))))))
 
+(flycheck-define-checker scheme-chicken
+  "A CHICKEN Scheme syntax checker using the CHICKEN compiler `csc'.
+
+See URL `http://call-cc.org/'."
+  :command ("csc" "-A" "-local" source)
```

Comment:
`-local` was suggested to me from CHICKEN users. Quoting: "-local enables Chicken to warn about e.g. fn.calls with bad arguments (bc it can assume the function won't be redefined)".

I'll use `-analyze-only` instead of  `-A`.